src/app/auth/components/ChangePassword/ChangePassword.tsx

```diff
@@ -64,15 +64,25 @@ export default function ChangePassword(props: Readonly<ChangePasswordProps>): JS
 
   const onUploadBackupKeyInputChanged = async (e) => {
     const file = e.target.files[0];
-    const backupKey = await file.text();
-    const isValidBackupKey = validateMnemonic(backupKey);
+    const backupKeyContent = await file.text();
```

Comment:
It may be better to rename this variable so that it does not match the state variable